hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -940,16 +991,17 @@ def stop_example(self, discard=False):
             self.observer.kill_branch()
 
     def note_event(self, event):
```

Comment:
Any suggestions for the type of `event`? this doesn't seem to be called

Reply:
[Based on the docs here](https://hypothesis.readthedocs.io/en/latest/details.html#hypothesis.event) I think we accept any `Hashable` object which we can convert to a string.